### ISSUE TITLE
feat: Image Classifier ('What's in this image?')

### ISSUE THREAD

**Issue by heysaiyad** [heysaiyad/dev-toolkit#205]
Build a tool where a user uploads an image, and a pre-trained model (like MobileNet via TensorFlow.js) attempts to classify the main subject of the image, providing its top predictions.

**Comment by techieadi4703** [comment]
I'm interested in this issue can you assign me this issue to me under hacktoberfest label.
